Yoast/wordpress-seo
# [How To] Set labels for time duration

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* @afercia advised to use a `<fieldset>` and `<legend>`, so I did.

## Test instructions

This PR can be tested by following these steps:

* Ideally @afercia should acceptance test this, but if not:
* Go to Safari, and verify that it recognizes the legend (Time needed), and the labels "hours" and "minutes" for the input fields.

## Quality assurance


Fixes #10750
